Engine/src/Engine/EngineUI/FrameBuffer.cpp

```diff
@@ -0,0 +1,52 @@
+#include "FrameBuffer.h"
```

Comment:
I would suggest leaving this class out of the PR since it's out of scope of this task.